greenguard/loader.py

```diff
@@ -70,7 +81,97 @@ def load(self, target=True):
             'turbines': self._read_csv(self._turbines),
         }
 
-        X = self._read_csv(self._target, True)
+        X = self._read_csv(self._target_times, True)
+        if return_target:
+            y = X.pop(self._target_column)
+            return X, y, tables
+
+        else:
+            return X, tables
+
+
+class GreenGuardRawLoader(object):
+    """GreenGuardRawLoader class.
+
+    The GreenGuardRawLoader class provides a simple interface to load a
+    timeseries dataset in raw format and return it in the format expected
+    by the GreenGuard Pipelines.
+
```

Comment:
The GreenGuardRawLoader class provides a simple interface to load a  time series data provided turbine in raw format and return it in the format expected by the GreenGuard Pipelines.